backend/src/main/java/wooteco/retrospective/presentation/attendance/AttendanceController.java

```diff
@@ -24,8 +33,27 @@ public AttendanceController(AttendanceService attendanceService) {
     @PostMapping("/time")
     public ResponseEntity<AttendanceResponse> postTime(@Valid @RequestBody AttendanceRequest attendanceRequest) {
         AttendanceResponse attendanceResponse = AttendanceResponse.of(
-                attendanceService.postAttendance(attendanceRequest));
+            attendanceService.postAttendance(attendanceRequest));
 
         return ResponseEntity.ok().body(attendanceResponse);
     }
+
+    @DeleteMapping("/time")
+    public ResponseEntity<Void> deleteTime(@Valid @RequestBody AttendanceRequest attendanceRequest) {
+        attendanceService.deleteAttendance(attendanceRequest);
+
+        return ResponseEntity.ok().build();
```

Comment:
```suggestion
        return ResponseEntity.noContent().build();
```

delete후 따로 반환하는 값이 없을 경우에는 204가 규약입니다. MDN을 참고해주세요!
https://developer.mozilla.org/ko/docs/Web/HTTP/Methods/DELETE

Reply:
저도 204 쓰고 싶었는데 api 명세에 200이더라구요..! 명세 < 규약??

Reply:
그 의미가 200을 준다기 보다는 200번대를 주라는 것으로 이해했는뎅... 204로 주더라도 프론트입장에서 status === 200 으로 체크하진 않아서 괜찮을거라 생각해요 @soulgchoi 그렇지..?